pkg/ebpf/c/tracee.bpf.c

```diff
@@ -335,6 +336,8 @@ enum container_state_e {
 #define NUMBER_OF_SYSCALLS_TO_CHECK_X86 18
 #define NUMBER_OF_SYSCALLS_TO_CHECK_ARM 14
 
+#define MAX_PATH_SIZE                   64
```

Comment:
This constant name is confusing. We already save pathanmes which are bigger than this.